src/test/kotlin/uk/gov/justice/digital/hmpps/pollpush/services/health/HealthCheckIntegrationTest.kt

```diff
@@ -48,14 +50,28 @@ class HealthCheckIntegrationTest : IntegrationTest() {
 
   @Test
   fun `Health ping page is accessible`() {
-    subPing(200)
```

Comment:
don't need to sub it as only calling `/health/ping` here and shouldn't touch dependencies